File: comms/bulk-messaging/messages/fetch-message-operation/fetch-message-operation.py
from twilio import TwilioComms

client = TwilioComms(
    account_id="<username>",
    auth_token="<password>",
)

client.messages.fetch_operation(
    operation_id="comms_operation_01h9krwprkeee8fzqspvwy6nq8",
)

File: comms/bulk-messaging/messages/fetch-message-operation/fetch-message-operation.php
<?php

namespace Example;

use Twilio\TwilioComms;

$client = new TwilioComms(
    accountId: '<username>',
    authToken: '<password>',
);
$client->messages->fetchOperation(
    'comms_operation_01h9krwprkeee8fzqspvwy6nq8',
);
